internal/cli/atlas/privateendpoints/gcp/interfaces/create.go

```diff
@@ -0,0 +1,115 @@
+	privateEndpointServiceID string
+	privateEndpointGroupID   string
+	gcpProjectID             string
+	endpoints                []string
+}
+
+func (opts *CreateOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *CreateOpts) parseEndpoints() []*atlas.GCPEndpoint {
+	endpoints := make([]*atlas.GCPEndpoint, len(opts.endpoints))
+	for i, endpoint := range opts.endpoints {
+		s := strings.Split(endpoint, "@")
+		endpoints[i] = &atlas.GCPEndpoint{
+			EndpointName: s[0],
+			IPAddress:    s[1],
+		}
+	}
+	return endpoints
+}
+
+var createTemplate = "Interface endpoint '{{.EndpointGroupName}}' created.\n"
+
+func (opts *CreateOpts) Run() error {
+	r, err := opts.store.CreateInterfaceEndpoint(opts.ConfigProjectID(), provider, opts.privateEndpointServiceID, opts.newInterfaceEndpointConnection())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newInterfaceEndpointConnection() *atlas.InterfaceEndpointConnection {
+	return &atlas.InterfaceEndpointConnection{
+		EndpointGroupName: opts.privateEndpointGroupID,
+		GCPProjectID:      opts.gcpProjectID,
+		Endpoints:         opts.parseEndpoints(),
+	}
+}
+
+// mongocli atlas privateEndpoint(s) gcp interface(s) create <endpointGroupId> --endpointServiceId endpointServiceId --gcpProjectId gcpProjectId --endpoint endpointName1@ipAddress1,...,endpointNameN@ipAddressN [--projectId projectId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:     "create <endpointGroupId>",
+		Aliases: []string{"add"},
+		Short:   "Create a GCP private endpoint interface.",
+		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":                "endpointGroupId",
+			"requiredArgs":        "endpointGroupId",
+			"endpointGroupIdDesc": "Unique identifier for the endpoint group.",
+		},
+		Example: `$ mongocli atlas privateEndpoints gcp interfaces create endpoint-1 --endpointServiceId 61eaca605af86411903de1dd --gcpProjectId mcli-private-endpoints --endpoint endpoint-0@10.142.0.2,endpoint-1@10.142.0.3,endpoint-2@10.142.0.4,endpoint-3@10.142.0.5,endpoint-4@10.142.0.6,endpoint-5@10.142.0.7`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.privateEndpointGroupID = args[0]
+			return opts.Run()
+		},
+	}
+	cmd.Flags().StringVar(&opts.privateEndpointServiceID, flag.EndpointServiceID, "", usage.EndpointServiceID)
+	cmd.Flags().StringVar(&opts.gcpProjectID, flag.GCPProjectID, "", usage.GCPProjectID)
+	cmd.Flags().StringSliceVar(&opts.endpoints, flag.Endpoint, []string{}, usage.LinkTokenAccessListCIDREntries)
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&opts.endpoints, flag.Endpoint, []string{}, usage.Endpoint)
```
The usage is wrong. Please, create the correct usage for this flag. It should contains an example with the separator `@`

Reply:
Copy-paste mistake... thanks for spotting that! I have fixed it in the latest commit.